dirty_cat/gap_encoder.py

```diff
@@ -514,6 +515,14 @@ class GapEncoder(BaseEstimator, TransformerMixin):
     max_iter_e_step : int, default=20
         Maximum number of iterations to adjust the activations h at each step.
 
+    handle_missing : 'error' or '' (default)
+        Whether to raise an error or impute with blank string '' if missing
```

Comment:
Maybe "passthrough" instead of ''

Reply:
Well I guess we're not passing through, as we're imputing the missing data. Am I missing something ?
Thanks :)

Reply:
Maybe "zeroout"?

I just don't love the empty string. It doesn't really tell what it does to a casual reader.

Reply:
I agree the empty string is not clear, maybe we can simply set `handle_missing = "impute"` ?

Reply:
OK, then "zero_impute". I don't like "impute", because people typically expect conditional imputations.

Reply:
I changed the default parameter to `zero_impute`, while still imputing with empty strings